test/integration/volumescheduling/volume_binding_test.go

```diff
@@ -1180,6 +1194,20 @@ func validatePVPhase(t *testing.T, client clientset.Interface, pvName string, ph
 	}
 }
 
+func waitForPVPhase(client clientset.Interface, pvName string, phase v1.PersistentVolumePhase) error {
+	return wait.Poll(time.Second, 1*time.Second, func() (bool, error) {
```

Comment:
use PollImmediate to avoid delaying one second on every call

Reply:
doesn't the second arg mean this times out after one attempt?

Reply:
fixed